fractal-puppy/Engine
# Resource Manager - Build without Assets folder

The Resource Manager has been updated to enable builds without the Asset folder. To achieve that .meta files have been updated and they are stored also in Library.

- To avoid errors the old metafiles are still accepted but they will automatically be updated once the Engine starts.
- It is highly recommended to delete the Library folder before using the engine with the update.
- Once the metafiles of the assets you are using have been updated you can upload them again to Git, just make sure that the UID is still the same.

*To test 1 - Debug:*
- Load the Graveyard scene and check that all the assets have been loaded.

*To test 2 - Game Build:*
- Change to "Game Build"
- Open "config.json" file and after "defaultsceneUID:" insert this number: 3063813660 (or the UID of the scene you will like to try).
- Compile everything.
- Copy "Game" folder to anywhere on your disk (Ex: Desktop).
- Copy the .exe file from the "GameBuild" folder into the "Game" folder you just copied.
- Double click on the .exe file and test that the game works.
